src/tool/cli.rs

```diff
@@ -28,7 +28,7 @@ pub struct Cli {
     /// Specify the path to the project root (If not provided uses current
     /// working directory)
     #[arg(long, short, global = true, value_name = "FOLDER")]
-    path: Option<String>,
+    pub path: Option<PathBuf>,
```

Comment:
Why do we need to make this pub? I don't see where we are using it outside. We really shouldn't be using it outside of this file thought as we set the directory here and it is meant to be transparent to the rest of the program.

Reply:
True, I made this change before realizing the WD is changed automatically and forgot to reset it